File: 6-Tarea-Arreglos-matrices/C.Omitiendo-el-entero-mas-grande.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int N = scanner.nextInt();
    int[] arr = new int[N];
    int max = -1;

    for (int i = 0; i < N; i++) {
      arr[i] = scanner.nextInt();

      if (arr[i] > max)
        max = arr[i];
    }

    int K = N;
    for (int i = 0; i < N; i++)
      if (arr[i] == max) K--;

    System.out.println(K);

    for (int i = 0; i < N; i++)
      if (arr[i] != max)
        System.out.print(arr[i] + " ");
  }
}

File: 6-Tarea-Arreglos-matrices/C.Omitiendo-el-entero-mas-grande.cs
using System.Collections.Generic;
using System.Linq;
using System;

class Program
{
  static void Main(string[] args)
  {
    int N = int.Parse(Console.ReadLine());
    int[] arr = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();
    int max = -1;

    for (int i = 0; i < N; i++) {
      if (arr[i] > max)
        max = arr[i];
    }

    int K = N;
    for (int i = 0; i < N; i++)
      if (arr[i] == max) K--;

    Console.WriteLine(K);

    for (int i = 0; i < N; i++)
      if (arr[i] != max)
        Console.Write(string.Format("{0} ", arr[i]));
  }
}